Scripts/script-TaniumMultiHash.yml

```diff
@@ -0,0 +1,184 @@
+commonfields:
+  id: d7e15b38-e402-4969-85b4-84958da63cc1
+  version: 59
+name: TaniumMultiHash
+script: |-
+  # Python template - reading arguments, calling a command, handling errors and returning results
+  import pandas as pd
+  from itertools import chain
+  def filter_values(contents):
```

Comment:
Could you please add documentation for this function?

Reply:
yes will do.